examples/showcase-all-resources.tf

```diff
@@ -0,0 +1,105 @@
+# App
+
+resource "ably-app" "app1" {
```

Comment:
Mentioned in the conversation. Use snake case for resource. e.g. ably_app

Reply:
Fixed in 505b3f0207c9aa3a9e2d07ee9554b84a7786a4e8